spotmarketrequest_test.go

```diff
@@ -0,0 +1,141 @@
+package packngo
+
+import (
+	"log"
+	"testing"
+	"time"
+)
+
+func TestAccSpotMarketRequestBasic(t *testing.T) {
+	// This test is only going to create the spot market request with
+	// max bid price set to half of current spot price, so that the devices
+	// are not run at all.
+	skipUnlessAcceptanceTestsAllowed(t)
+
+	c, projectID, teardown := setupWithProject(t)
+	defer teardown()
+
+	ps := InstanceParameters{
+		BillingCycle:    "hourly",
+		Plan:            "baremetal_0",
+		OperatingSystem: "rancher",
+		Hostname:        "test{{index}}",
+	}
+
+	prices, _, err := c.SpotMarket.Prices()
+	pri := prices["ewr1"]["baremetal_0"]
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cr := SpotMarketRequestCreateRequest{
+		DevicesMax:  3,
+		DevicesMin:  2,
+		FacilityIDs: []string{"ewr1", "ewr1"},
```

Comment:
Does it really allow to pass two same facility IDs?

Reply:
yeah 2 same facilities go through. I changed it to different ones for the sake of example. The API semantics should be reported but probably beyond scope of this PR.